.github/workflows/deploy.yml

```diff
@@ -214,4 +214,6 @@ jobs:
           environment_name: ${{ secrets.TF_PROJECT_NAME}}-${{ needs.set_environment.outputs.env_name }}-environment
           region: ${{ secrets.TF_AWS_REGION }}
           version_label: ${{ github.sha }}-${{ github.run_id }}-${{ github.run_attempt }}
-          deployment_package: infrastructure/source_bundle/deploy.zip
\ No newline at end of file
+          deployment_package: infrastructure/source_bundle/deploy.zip
+          wait_for_deployment: true
```

Comment:
This option is true by default, but I think it's good to leave this here for verbosity